avaframe/com1DFA/com1DFA.py

```diff
@@ -1700,6 +1763,53 @@ def readPartFromPickle(inDir, flagAvaDir=False):
     return Particles, TimeStepInfo
 
 
+def mainTrackParticles(ParticlesList, TimeStepInfo, center, radius, particleProperties):
```

Comment:
- TimeStepInfo is missing from the docstring
- please rename variable to particleList and timeStepInfo